src/win64.mak

```diff
@@ -13,7 +13,7 @@ OS=windows
 ################################### Rules ####################################
 
 .d.exe:
-	$(HOST_DC) -of$@ $<
+	$(HOST_DC) -g -of$@ $<
```

Comment:
Temporary change?

Reply:
Well, I added this because it was very helpful when debugging the build failures and realized that there's no reason not to build the build tools with debug information as otherwise the next person to debug a failure is hit an unhelpful stack trace again, so I left this.

Reply:
Ok.